WISE-Community/WISE-Client
# TinyMCE UI changes

Added a custom tinymce skin (https://www.tiny.cloud/docs/advanced/creating-a-skin/) and set select button label width to 'auto' so select buttons take up less space in the toolbar. Also moved the insert options earlier in the toolbar.

To test:
- Open a unit in the authoring tool and open a step with an HTML component.
- Verify that toolbar changes are shown and tinymce editing works.

Resolves #91.
